components/resources/brave_components_strings.grd

```diff
@@ -550,50 +550,8 @@
       <message name="IDS_BRAVE_REWARDS_LOCAL_ADS_CONFIRMATION_TYPE_VIEW" desc="">Viewed</message>
 
       <message name="IDS_BRAVE_REWARDS_LOCAL_REDIRECT_MODAL_CLOSE" desc="">Close</message>
-      <message name="IDS_BRAVE_REWARDS_LOCAL_REDIRECT_MODAL_DEVICE_LIMIT_REACHED_TEXT" desc="">Your Brave Rewards cannot be verified because you've reached the maximum verified device limit.</message>
```

Comment:
These strings have been moved into `rewards_strings.grdp` and renamed to match the key used on the front-end.

Reply:
Wherever it was moved, we should update this to say:

>You've reached the maximum device limit for this account.